blockchain/My-Wallet-V3-iOS
# Merge branch - kyc into dev

## Objective

Merges all the changes from the `kyc` branch into `dev`.

## Description

Atm we are working off of two different branches: `kyc` and `dev`. `kyc` initially branched off of `dev` but now have gone out of sync since there were some changes that have been introduced to both branches. To prevent some hard-to-fix merge conflicts, this change merges `kyc` back into `dev`. Once this is merged, all KYC features should be rebased against `dev`.

## How to Test

Pull the `new_dev` branch locally, test the app and make sure that all compiles/works.

## Screenshot

N/A

## Related Information

* Ticket Number: N/A

## Merge Checklist


...note that unit tests are failing but I don't think this is because of a failed merge (could be a bug in `kyc`). Will need to dig into it more
